### ISSUE TITLE
tf.py_func is deprecated in TF V2. Instead

### ISSUE THREAD

**Issue by GbengaOdesanmi** [openai/baselines#832]
Please ho do i solv this problem python thanks in advance -m baselines.run --alg=ppo2 --env=CartPole-v0 --network=mlp --num_timesteps=200
Logging to /tmp/openai-2019-02-22-03-06-34-542169
Logging to /tmp/openai-2019-02-22-03-06-34-638706
env_type: classic_control
2019-02-22 03:06:34.979205: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-02-22 03:06:34.981967: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 3591945000 Hz
2019-02-22 03:06:34.982229: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x5585dec13690 executing computations on platform Host. Devices:
2019-02-22 03:06:34.982246: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
Training ppo2 on classic_control:CartPole-v0 with arguments 
{'network': 'mlp'}
WARNING:tensorflow:From /home/gbenga/baselines/baselines/common/input.py:57: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
WARNING:tensorflow:From /home/gbenga/baselines/baselines/common/models.py:50: flatten (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.flatten instead.
WARNING:tensorflow:From /home/gbenga/Downloads/abiona1008/envs/tensorflow/lib/python3.7/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
WARNING:tensorflow:From /home/gbenga/Downloads/abiona1008/envs/tensorflow/lib/python3.7/site-packages/tensorflow/python/ops/math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
WARNING:tensorflow:From /home/gbenga/Downloads/abiona1008/envs/tensorflow/lib/python3.7/site-packages/tensorflow/python/ops/math_grad.py:102: div (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Deprecated in favor of operator or tf.math.divide.
WARNING:tensorflow:From /home/gbenga/baselines/baselines/common/mpi_adam_optimizer.py:25: py_func (from tensorflow.python.ops.script_ops) is deprecated and will be removed in a future version.
Instructions for updating:
tf.py_func is deprecated in TF V2. Instead, use
    tf.py_function, which takes a python function which manipulates tf eager
    tensors instead of numpy arrays. It's easy to convert a tf eager tensor to
    an ndarray (just call tensor.numpy()) but having access to eager tensors
    means `tf.py_function`s can use accelerators such as GPUs as well as
    being differentiable using a gradient tape.
